{"uuid": "26c248c5-22df-4905-b20d-886618ef5633", "children": ["b2906ecb-53b8-49dd-ae27-2936068660b9"], "befores": [{"name": "get_instance", "status": "broken", "statusDetails": {"message": "TypeError: 'module' object is not callable\n", "trace": "  File \"D:\\学习\\todd_study\\venv\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\学习\\todd_study\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\学习\\todd_study\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\HogwartsSDET17_todd\\testing\\test_Calculator.py\", line 22, in get_instance\n    calc: Calculator = Calculator()\n"}, "start": 1612410765985, "stop": 1612410765985}], "start": 1612410765985, "stop": 1612410766006}